pandas/tests/frame/methods/test_astype.py

```diff
@@ -660,6 +656,15 @@ def test_astype_dt64_to_string(self, frame_or_series, tz_naive_fixture, request)
         alt = obj.astype(str)
         assert np.all(alt.iloc[1:] == result.iloc[1:])
 
+    def test_astype_td64_to_string(self, frame_or_series):
```

Comment:
can you parametersize over td, dt, dt w/tz & period

Reply:
not nicely; the dt64 one uses tz_naive_fixture

Reply:
ok can you add that as another test then. just to make sure coverng the bases.

Reply:
add what as another case?  we have the td64 case here and the dt64 case right above here

Reply:
ahh ok. 

period covered? if not pls add (followup ok)